packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -93,4 +105,5 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   }
 
   setHeadComponents(headComponents)
+  return true
```

Comment:
Any reason this was added? it's not needed so not sure it needs to be here.

Reply:
Added it due to the eslint rule consistent-return complained, because of this line https://github.com/gatsbyjs/gatsby/blob/4844244b2e1579ea2833667a576301fb88e661f8/packages/gatsby-plugin-manifest/src/gatsby-ssr.js#L17

Any suggestion?

Reply:
Ah, I missed the other return got added. If that need to be there that's fine. Thanks